src/components/blankSlate/examples/BlankSlateExample.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { BlankSlate } from '../BlankSlate';
+
+export class BlankSlateExample extends React.Component<any, any> {
+
+  render() {
+    return (
+      <div className='mt2'>
+        <h1 className='text-blue mb1'>BlankSlate List</h1>
+        <div className='form-group'>
+          <label className='form-control-label'>Default BlankSlate</label>
+          <BlankSlate />
+        </div>
+        <div className='form-group'>
+          <label className='form-control-label'>BlankSlate with title</label>
+          <BlankSlate title='Title test' />
+        </div>
+        <div className='form-group'>
+          <label className='form-control-label'>BlankSlate with title</label>
+          <BlankSlate title='description test' />
```

Comment:
It should probably be `BlankSlate with description` and description instead of title